docs/topics/accessing-unmatched-rules.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="accessing-unmatched-rules_{context}"]
+= Accessing unmatched rules
+
+To access unmatched rules, you must run the analysis with enhanced logging enabled.
+
+. Navigate to *Advanced* under *Application analysis*.
+. Select *Options*.
+. Check *Enhanced advanced analysis details*.
+
+When you run an analysis:
+
+. Navigate to analysis details and select the `issues.yaml` file.
```

Comment:
Hi @anarnold97, I suggest we clarify the navigation path to avoid any potential confusion for the user. Specifically, we should guide them to first navigate to "Reports" in the side drawer, then click on "View analysis details," which opens the YAML/JSON format log view. From there, the user can select the issues.yaml file from the dropdown menu.
![Screenshot from 2024-08-12 17-33-11](https://github.com/user-attachments/assets/60712870-9e2c-4605-a5ad-a47c34eda6f6)
![Screenshot from 2024-08-12 17-19-21](https://github.com/user-attachments/assets/262d58d1-8e41-4844-b923-3ad273cb6ba5)

Reply:
@kpunwatk - is this updated text what you meant? Thanks

Reply:
Yes @anarnold97 not sure but we could update this line for clarity ```Navigate to analysis details and select the `issues.yaml` file.```